lib/usage.js

```diff
@@ -96,11 +96,23 @@ module.exports = function (yargs, y18n) {
     epilog = msg
   }
 
-  var wrap = windowWidth()
+  var wrapSet
+  var wrap
   self.wrap = function (cols) {
+    wrapSet = true
     wrap = cols
   }
 
+  function getWrap () {
+    // lazily call windowWidth() because it's very expensive,
+    // and only needs to be called if the user wants to show usage/help
+    if (!wrapSet) {
+      wrap = windowWidth()
+    }
+    wrapSet = true
```

Comment:
Can we make this the first line under the `if (!wrapSet)` condition instead? Seems equivalent but more efficient that way.